lib/segment/analytics/worker.rb

```diff
@@ -44,6 +45,7 @@ def run
             end
           end
 
+          logger.debug("Sending request for #{@batch.length} items")
           res = @request.post(@write_key, @batch)
```

Comment:
hey @rohitpaulk, as a follow up; let's log the result of the HTTP response. I think the status code might be helpful to have here.